04_HowTos/QueryClient/How_to_query_DESI_DR1_Data.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Optional comment because one can infer this pretty easily, but for extra clarity you could add a note that night is written as <code>YYYYMMDD</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good idea.</p>